src/forms/CheckboxGroup.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react"
+
+import "./CheckboxGroup.scss"
+import "../actions/Button.scss"
+import { ButtonProps } from "actions/Button"
+
+export interface CheckboxGroupProps extends Pick<ButtonProps, "size" | "variant"> {
+  /** Label content */
+  label?: string
+  /** Current selected values*/
+  values: string[]
+  /** An array of strings representing each item in the group*/
+  options: string[]
+  /** Element ID */
+  id: string
+  /** Additional CSS classes */
+  className?: string
+  /** ID for selecting in tests */
+  testId?: string
+  /** function to call when a checkbox is clicked*/
+  onChange: (values: string[]) => void
+  /** Appearance of the checked input*/
+  checkedVariant?:
+    | "primary"
+    | "primary-outlined"
+    | "secondary"
+    | "secondary-outlined"
+    | "success"
+    | "success-outlined"
+    | "alert"
+    | "alert-outlined"
+    | "highlight"
+    | "highlight-outlined"
+    | "text"
+}
+
+const CheckboxGroup = (props: CheckboxGroupProps) => {
+  const classNames = ["seeds-checkbox-group"]
+  if (props.className) classNames.push(props.className)
+
+  const handleCheckboxChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, checked } = event.target
+    const newValue = checked ? [...props.values, name] : props.values.filter((v) => v !== name)
+    props.onChange(newValue)
+  }
+
+  return (
+    <div id={props.id} className={classNames.join(" ")} data-testid={props.testId}>
+      {props.options.map((option) => (
+        <div key={`${option}-container`}>
+          <input
+            type="checkbox"
+            name={option}
+            id={option}
```

Comment:
Each input ID needs to be unique across the entire web page, so wondering what's the best way to enforce it's unique. Maybe a compound key that also takes the overall component ID into account?